src/controller/python/chip/yaml/runner.py

```diff
@@ -254,7 +279,9 @@ def __init__(self, test_step):
         args = test_step.arguments['values']
         request_data_as_dict = Converter.convert_list_of_name_value_pair_to_dict(args)
 
-        self._expire_existing_session = request_data_as_dict.get('expireExistingSession', False)
+        # Only if expire existing session is explicitly false do we want to supress expiring
+        # the session.
+        self._expire_existing_session = request_data_as_dict.get('expireExistingSession', True)
```

Comment:
Wait, why this change?  The comment should explain _why_ we want the behavior, not what the behavior is.